_config.yml

```diff
@@ -8,8 +8,8 @@ description: Java Design Patterns
 
 # ! Attention: the lines starting with "baseurl" and "url" are edited on travis build with a search and replace regex
 # Any Change in those two lines should be checked with the "cibuild" script
-baseurl: "/java-design-patterns" # the subpath of your site, e.g. /blog/
-url: "https://iluwatar.github.io" # the base hostname & protocol for your site
+baseurl: "" # the subpath of your site, e.g. /blog/
+url: "http://localhost:4000" # the base hostname & protocol for your site
```

Comment:
why this change? with `jekyll serve` in the java-design-patterns directory it should display the site correctly including all links, because they are relative. Jekyll serves the site under `http://127.0.0.1:4000/java-design-patterns/`